vendor/openvino/openvino.BUILD

```diff
@@ -207,9 +193,14 @@ cc_library(
             "inference-engine/src/inference_engine/os/lin/*.cpp",
         ]),
     }),
+    hdrs = glob([
+        # "inference-engine/src/readers/ir_reader/ie_ir_version.hpp",  # TODO: Ok to remove?
```

Comment:
Here (and I think a few other places in this BUILD file) I seem to not be listing any *.h or *.hpp files from the directory `inference-engine/src/readers/ir_reader` (in this case) and everything builds fine; however, if I remove that directory from the `includes` list the build breaks. I'm confused as to how that is possible.